Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/packages/passphrase.adoc

= Lisk Elements Passphrase
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference for the Pasphrase package of Lisk Elements. It copmrises usage examples, available options and example responses.

Reviewer: ```suggestion
:description: Technical references regarding the Passphrase package of Lisk Elements. This consists of usage examples, available options and example responses.
```
